posthog-node/src/feature-flags.ts

```diff
@@ -4,8 +4,162 @@ export const getLocalFeatureFlag = (
   allFeatureFlags: PostHogFeatureFlag[],
   key: string,
   distinctId: string,
-  groups?: Record<string, string> | undefined
+  groups?: Record<string, string> | undefined,
+  personProperties?: Record<string | number, any>,
+  groupProperties?: Record<string | number, any>
 ) => {
   // TODO: Evaluate flags purely (no state) to ensure easy testing
-  return null
+  const featureFlag = allFeatureFlags.find(featureFlag => featureFlag.key === key)
+  if (!featureFlag) {
+    return null
+  }
+  if (featureFlag.ensure_experience_continuity) {
+    throw new Error('Flag has experience continuity enabled')
```

Comment:
Might want to add a more clarifying message, why such feature flag aren’t allowed for local evaluation